README.md

```diff
@@ -45,6 +45,80 @@ In order to track which migrations have been ran, `cfmigrations` needs to instal
 
 If you find a need to, you can uninstall the migrations table by calling the `uninstall()` method or by running `migrate uninstall` from `commandbox-migrations`. Running this method will rollback all ran migrations before dropping the `cfmigrations` table.
 
+### Configuration
+
+The module may be configured with a single Migration Service or multiple migration services with different managers.  The default manager for the Application is `QBMigrationManager`, but you may use others, such as those included with the `cbmongodb` and `cbelasticsearch` modules or roll your own.
+
+The default configuration for the module settings are:
+
+```
+moduleSettings = {
+    "cfmigrations" : {
```

Comment:
I'm not convinced we need to support this single manager syntax.  Why not just have a `default` manager?  Then our code can just deal with named managers all the time.

Reply:
So are you thinking that a key of `default` becomes the DSL `MigrationServce@cfmigrations` or `MigrationService:default` .  This will affect how we update the commandbox module.

Reply:
I think `MigrationService:default`.  I like having just one set of settings we have to parse.  This will be a breaking change for `cfmigrations` (`v3.0.0`) and `commandbox-migrations` can follow suit in `v4.0.0`.

Reply:
Will do!

Reply:
How would you feel about having the following settings structure, then:

```
settings = {
			"default" : {
				"manager" : "cfmigrations.models.QBMigrationManager",
				"properties" : {
					"defaultGrammar" : "BaseGrammar"
				}
			}
			
		};
```

having a top-level `manager` key seems redundant, when all of the keys below will need a `manager` key.  Since all of the settings pass through to the migration service instantiated, we can just loop the `keyArray()` of the settings instead of `settings.manager`

Reply:
I think I would have named the key `managers` (plural).  having the key does seem slightly redundant.  On the other hand, it allows us to add global settings in the future if the need arises.  I'm not convinced either way right now.

Reply:
Done